Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1093,20 +1093,23 @@ This section elaborates on the various ``kubeProxyReplacement`` options:
   performed by Cilium in a more efficient manner with the help of eBPF instead of
   allocating and binding actual sockets.
 
-- ``kubeProxyReplacement=partial``: Similarly to ``probe``, this option is
-  intended for a hybrid setup, that is, kube-proxy is running in the Kubernetes cluster
-  where Cilium partially replaces and optimizes kube-proxy functionality. As opposed to
-  ``probe`` which checks the underlying kernel for available eBPF features and automatically
-  disables components responsible for the eBPF kube-proxy replacement when kernel support
-  is missing, the ``partial`` option requires the user to manually specify which components
-  for the eBPF kube-proxy replacement should be used. When ``kubeProxyReplacement``
-  is set to ``partial`` make sure to also set ``enableHealthCheckNodeport`` to
-  ``false``, so that the Cilium agent does not start the NodePort health check server.
-  Similarly to ``strict`` mode, the Cilium agent will bail out on start-up with an error
-  message if the underlying kernel requirements are not met. For fine-grained configuration,
-  ``hostServices.enabled``, ``nodePort.enabled``, ``externalIPs.enabled``
-  and ``hostPort.enabled`` can be set to ``true``. By default all four options are set
-  to ``false``. A few example configurations for the ``partial`` option are provided below.
+- ``kubeProxyReplacement=partial``: Similarly to ``probe``, this option is intended for
+  a hybrid setup, that is, kube-proxy is running in the Kubernetes cluster where Cilium
+  partially replaces and optimizes kube-proxy functionality. As opposed to ``probe``
+  which checks the underlying kernel for available eBPF features and automatically
+  disables components responsible for the eBPF kube-proxy replacement when kernel
+  support is missing, the ``partial`` option requires the user to manually specify
+  which components for the eBPF kube-proxy replacement should be used.  Similarly to
+  ``strict`` mode, the Cilium agent will bail out on start-up with an error message if
+  the underlying kernel requirements are not met. For fine-grained configuration,
+  ``hostServices.enabled``, ``nodePort.enabled``, ``externalIPs.enabled`` and
+  ``hostPort.enabled`` can be set to ``true``. By default all four options are set to
+  ``false``. If you are setting ``nodePort.enabled`` to true, make sure to also
+  set ``nodePort.enableHealthCheck`` to ``false``, so that the Cilium agent does not
+  start the NodePort health check server. (``kube-proxy`` will also attempt to start
+  this server, and there will be a clash when cilium attempts to bind its server to the
+  same port.) A few example configurations for the ``partial`` option are provided
```

Comment:
Small nit: I'd rather have the sentence closing after the parenthesis, given that it expands on `nodePort.enableHealthCheck`.
```suggestion
  start the NodePort health check server (``kube-proxy`` will also attempt to start
  this server, and there would otherwise be a clash when cilium attempts to bind its server to the
  same port). A few example configurations for the ``partial`` option are provided
```